pkg/agent/openflow/feature.go

```diff
@@ -0,0 +1,280 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openflow
+
+import (
+	binding "antrea.io/antrea/pkg/ovs/openflow"
+)
+
+// OVS pipelines are generated by a framework called FlexiblePipeline. There are some abstractions introduced in this
```

Comment:
@jianjuns @tnqn From line 21 to line 103 is newly updated.

Reply:
Looks clearer to me